apps/omg_watcher_rpc/priv/swagger/security_critical_api_specs.yaml

```diff
@@ -391,6 +391,12 @@ paths:
                                           type: string
                                         amount:
                                           type: integer
+                                        root_chain_txhash:
+                                          type: string
+                                        spending_txhash:
```

Comment:
Not sure are we emitting this `spending_txhash` yet?

Reply:
Added in anticipation of Med's recently merged PR: https://github.com/omisego/elixir-omg/pull/1492